provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -1254,7 +1254,8 @@
         },
         "timestamp": {
           "description": "The time at which the schedule should run, in ISO 8601 format. Eg: 2020-01-01T00:00:00Z. If you are supplying this, do not supply scheduleCron.",
-          "type": "string"
+          "type": "string",
+          "willReplaceOnChanges": true
```

Comment:
I don't think this is a real property that has any effect on the generated SDK. `replaceOnChanges` does, but we don't actually need it anymore since we're handling it in the provider code.

Reply:
Turns out `willReplaceOnChanges` is a [real thing](https://www.pulumi.com/docs/iac/packages-and-automation/pulumi-packages/schema/#:~:text=a%20update%20operation.-,willReplaceOnChanges,-boolean) used as documentation to indicate that the provider handles replacement.